a8c-files.php

```diff
@@ -69,7 +61,37 @@ function __construct() {
 		add_filter( 'option_upload_url_path', array( $this, 'upload_url_path' ), 10, 2 );
 	}
 
-	function use_jetpack_photon() {
+	private function init_jetpack_photon_filters() {
+		// The files service has Photon capabilities, but is served from the same domain.
+		// Force Jetpack to use it instead of the default Photon domains (`i*.wp.com`).
+		add_filter( 'jetpack_photon_domain', function() {
+			return home_url();
+		} );
+
+		// The sizes metadata is not used and mostly useless on Go so let's strip it out.
```

Comment:
Does unsetting it, versus setting it to an empty array, have any consequences? It'll surely break `srcset` support, which was likely broken to begin with given the uselessness of the data.